src/index.js

```diff
@@ -87,6 +87,73 @@ function displayAllData() {
   displaySleepChart();
 }
 
+$('#activity-submit').click(postNewActivityData);
+$('.hydration-button').click(postNewHydrationData);
+$('.sleep-button').click(postNewSleepData);
+
+function postNewActivityData(e) {
+  e.preventDefault();
+  fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/activity/activityData', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      userID: randomId,
```

Comment:
does this get another, different ID? would it be better to  use user.id?

Reply:
oh, never mind. I believe this is referencing the variable representing the id retrieved when we instantiate the user on page load, not calling the function again. While I still think I might prefer using user.id here, I think this should work fine.

Reply:
I agree on the user.id but for some reason the fetch wasn't finding user instance.  randomId worked though so I went with that.